samcli/cli/main.py

```diff
@@ -74,10 +74,6 @@ def print_cmdline_args(func):
     """
 
     def wrapper(*args, **kwargs):
-        if kwargs.get("config_file") and kwargs.get("config_env"):
```

Comment:
It was inaccurate. When the template file was used outside of the project source directory, instead of giving a warning that the config was not found, it output `samconfig.toml` which created an illusion that the file was found.

Reply:
I agree with your point that it gives an impression that the file is found. Here it gives the debugging information about the config file and config env. If the file is not found, the debug will show up later. I suggest changing the log to better describe this. Or combine it into the expanded cli parameters.

Reply:
I deleted this code because I added file directory echo when config file is parsed and because the config env name was [already in debug](https://github.com/aws/aws-sam-cli/blob/4e1a9796d9538e37dacc170d550c4f5561e10f63/samcli/cli/cli_config_file.py#L71). It didn't get to output without `SAM_DEBUG` but now it does.